ISSUES.md

```diff
@@ -2,7 +2,7 @@ If you open a GitHub Issue, here is our policy:
 
 1.  It must be a bug/performance issue or a feature request or a build issue or
     a documentation issue (for small doc fixes please send a PR instead).
-1.  Make sure the Issue Template is filled out.
+1.  Make sure the [Issue Template](./ISSUE_TEMPLATE.md) is filled out.
```

Comment:
I don't think we should link to this. We have multiple issue templates, the one you link to is not in real use and should probably be deleted (I'll take care of it)

Reply:
The file is now removed, so the PR needs to be updated